UDPClientGUI/ViewModels/ShellViewModel.cs

```diff
@@ -0,0 +1,98 @@
+﻿using Caliburn.Micro;
+using Sice.PoC.UDPCommGUI;
+using System.Configuration;
+using System.Net;
+
+namespace UDPCommGUI.ViewModels
+{
+    class ShellViewModel : Screen, IHandle<ClientStatusChangedEvent>
+    {
+        private readonly IEventAggregator _eventAggregator;
+        private string _iP;
+        private string _port;
+        private string _message;
+        private string _connectionStatus;
+
+        public ShellViewModel(IEventAggregator eventAggregator, SiceUDPClient siceUDPClient)
+        {
+            _eventAggregator = eventAggregator;
+            _eventAggregator.Subscribe(this);
+            IP = ConfigurationManager.AppSettings["IPAddress"];
+            Port = ConfigurationManager.AppSettings["PortNumber"];
+            ConnectionStatus = "Disconnected";
+        }
+
+        public string IP
+        {
+            get => _iP;
+            set
+            {
+                _iP = value;
+                NotifyOfPropertyChange(nameof(IP));
+            }
+        }
+
+        public string Port
+        {
+            get => _port;
+            set
+            {
+                _port = value;
+                NotifyOfPropertyChange(nameof(Port));
+            }
+        }
+
+        public string ConnectionStatus
+        {
+            get => _connectionStatus;
+            set
+            {
+                _connectionStatus = value;
+                NotifyOfPropertyChange(nameof(ConnectionStatus));
+            }
+        }
+
+        public string Message
+        {
+            get => _message;
+            set
+            {
+                _message = value;
+                NotifyOfPropertyChange(nameof(Message));
+            }
+        }
+
+        public bool CanConnect(string iP, string port)
+        {
+            if (!IPAddress.TryParse(iP, out IPAddress parseIP)) return false;
+            if (!int.TryParse(port, out int parsePort)) return false;
+            return true;
```

Comment:
Nothing wrong with this code (except I'd keep the braces in). 

You can write this as:
``` return IPAddress.TryParse(iP, out IPAddress parseIP) && int.TryParse(port, out int parsePort)```

If you wanted more readability you could do:
```
var validIP = IPAddress.TryParse(iP, out IPAddress parseIP);
var validPort = int.TryParse(port, out int parsePort);
return validIP && validPort;
```

The principles here are:
- Consider combining if expressions to reduce complexity.
- Consider using ternary operators or single statements to simplify expressions
- Though the second version is the same number of lines, it's easier to read, it's almost like written language.

Reply:
Adopted the second style